Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:14.10
+
+WORKDIR /vpn
+
+RUN apt-get install -y curl
+RUN curl -sL http://nginx.org/keys/nginx_signing.key | apt-key add -
+RUN echo "deb http://nginx.org/packages/ubuntu/ utopic nginx" >> /etc/apt/sources.list && \
+    echo "deb-src http://nginx.org/packages/ubuntu/ utopic nginx" >> /etc/apt/sources.list
+RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends \
+    openvpn \
+    iptables \
+    traceroute \
+    dnsutils \
+    nginx
```

Comment:
heh it's been a SUPER long time since I did anything like this so my review is probably not going to be great